company_website/static/common/scripts/external-links-handler.js

```diff
@@ -1,10 +1,15 @@
+let exception = "#talk-with-adrian"
+
 $('a').each(function() {
-   let local_link_expression = new RegExp('/' + window.location.host + '/');
-   if(!local_link_expression.test(this.href)) {
-       $(this).click(function(event) {
-           event.preventDefault();
-           event.stopPropagation();
-           window.open(this.href, '_blank');
-       });
-   }
+    if(!$(this).is(exception)) {
+        let local_link_expression = new RegExp('/' + window.location.host + '/');
```

Comment:
In javascript is convention to name variables using camel case instead of snake case. Please change variables in this function

Reply:
Done